clang-tools-extra/clang-tidy/hooks/GuardInForCheck.cpp

```diff
@@ -9,6 +9,7 @@
 #include "GuardInForCheck.h"
 #include "clang/ASTMatchers/ASTMatchFinder.h"
 #include <assert.h>
+#include <iostream>
```

Comment:
What is this for? IMHO the checks can use the LLVM framework for all their output needs...